exercises/storeWithArrays.js

```diff
@@ -48,19 +40,27 @@ function newSofa(name, price) {
  * and `false` otherwise.
  */
 function userCanAffordSofa(user, sofa) {
-  // This is your job. :)
+if (sofa[1] <= user[2]){
```

Comment:
Since `sofa[1] <= user[2]` evaluates to `true` or `false` directly, you can do

```javascdript
function userCanAffordSofa(user, sofa) {
  return sofa[1] <= user[2];
}
```